omnikinverter/omnikinverter.py

```diff
@@ -179,6 +179,9 @@ async def inverter(self) -> Inverter:
         if self.source_type == "html":
             data = await self.request("status.html")
             return Inverter.from_html(data)
+        if self.source_type == "cgi":
+            data = await self.request("inverter.cgi", params={"t": "123"})
```

Comment:
What's this `123` param for?

Reply:
No idea :)
The Web UI sends the following request:
```
var inverter = new XMLHttpRequest();
inverter.open("get", "inverter.cgi?t=" + Math.random(), true);
```

I figured the value for t is not important so kept it constant.
I can change it to send in a random number to more closely follow the Web UI if you like.